### PR TITLE
replace github.com/yyuu with github.com/pyenv

### PR DESCRIPTION
**Pull request by menduo**
replace github.com/yyuu with github.com/pyenv

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## 1.0.9
 
-* pyenv: Migrate project site from https://github.com/yyuu/pyenv to https://github.com/pyenv/pyenv
+* pyenv: Migrate project site from https://github.com/pyenv/pyenv to https://github.com/pyenv/pyenv
```
Comment (thomasjo) [inline CHANGELOG.md]: The change on this specific line doesn't make sense — it needs to be reverted. Read what it says, and you'll see why.

Comment (menduo) [inline CHANGELOG.md]: yes, my mistake.

Comment (yyuu) [inline CHANGELOG.md]: Please fix a mistake in `CHANGELOG.md` when you get a chance. Then I will merge this. Thanks for contribution~

**Comment by menduo** [comment]
@yyuu done.

**Comment by yyuu** [comment]
Thanks!

**Pull request merged**